src/plugins/workspace/public/components/workspace_list/index.tsx

```diff
@@ -72,12 +107,14 @@ export const WorkspaceList = ({ registeredUseCases$ }: WorkspaceListProps) => {
 
   const [pagination, setPagination] = useState({
     pageIndex: 0,
-    pageSize: 5,
+    pageSize: 10,
     pageSizeOptions: [5, 10, 20],
   });
   const [deletedWorkspaces, setDeletedWorkspaces] = useState<WorkspaceAttribute[]>([]);
   const [selection, setSelection] = useState<WorkspaceAttribute[]>([]);
   const [allDataSources, setAllDataSources] = useState<DataSourceAttributesWithWorkspaces[]>([]);
+  //  default workspace state
+  const [defaultWorkspace, setDefaultWorkspace] = useState<string | undefined>(undefined);
```

Comment:
Nit: Would it be more clear if it is named as `defaultWorkspaceId`?

Reply:
thanks, will update to `defaultWorkspaceId`